src/js/views/Planet.js

```diff
@@ -51,13 +259,19 @@ class Planet {
     this.uqmPlanetSeedChoices = [];
     this.uqmPlanetSeedChoice = "NONE";
 
-    // this.waterLevel = 0.5;
+    this.uqmGenerationTable = new UqmGenerationTable();
```

Comment:
`UqmPlanetTable` and `UqmGenerationTable` now have some overlapping responsibilities, but we can address that later. I'd love to have this PR merged asap. 🙂